tests/e2e/features/artifacts/depositPage/tokens.feature

```diff
@@ -0,0 +1,91 @@
+      | RPL       | 0x1CF855...5BC | https://firebasestorage.googleapis.com/v0/b/token-library.appspot.com/o/rpl.svg?alt=media&token=d00177fb-503a-45fe-bd50-dea7a4b7c32e                                                                                                             |
+      | UFI       | 0xa0C1BC...dAE | https://firebasestorage.googleapis.com/v0/b/token-library.appspot.com/o/ufi.svg?alt=media&token=1e2c543a-e2ff-4598-8722-f6fc9556b62d                                                                                                             |
+      | cbETH     | 0x75Af29...ED5 | https://firebasestorage.googleapis.com/v0/b/token-library.appspot.com/o/cbeth.svg?alt=media&token=e4bf7157-cc82-4300-aa6f-8d0a0dc044f7                                                                                                           |
+      | RAISE     | 0x3D79F1...46f | https://firebasestorage.googleapis.com/v0/b/token-library.appspot.com/o/raise.svg?alt=media&token=79740de5-ac54-437b-a982-aec0a5b62361                                                                                                           |
+      | LSD       | 0x458A2E...bD4 | https://firebasestorage.googleapis.com/v0/b/token-library.appspot.com/o/lsd.svg?alt=media&token=8f20e143-4862-457e-9d7c-36da9676845f                                                                                                             |
+      | ETHx      | 0x668cc2...39c | https://firebasestorage.googleapis.com/v0/b/token-library.appspot.com/o/ethx.svg?alt=media&token=362fe1c5-afb4-47cf-b766-c27aa11bde89&_gl=1*148cy0l*_ga*MTQ3NTc0NTE3Ni4xNjg2MDM4NzMy*_ga_CW55HF8NVT*MTY4NjYzODQ4Ny4xMy4xLjE2ODY2Mzg1NTguMC4wLjA. |
+      | WBTC      | 0xBBeB51...011 | https://firebasestorage.googleapis.com/v0/b/token-library.appspot.com/o/wbtc.svg?alt=media&token=502e8eee-d36a-4460-a97b-dad3e387cddd                                                                                                            |
+      | KNC       | 0x6ee46C...3e6 | https://firebasestorage.googleapis.com/v0/b/token-library.appspot.com/o/wbtc.svg?alt=media&token=502e8eee-d36a-4460-a97b-dad3e387cddd                                                                                                            |
+      | BEL       | 0xB83CFB...2D9 | https://firebasestorage.googleapis.com/v0/b/token-library.appspot.com/o/bel.png?alt=media&token=f8bbcfc2-6037-4626-870b-6a3b6630554b                                                                                                             |
+      | ZZ        | 0x1ab721...184 | https://firebasestorage.googleapis.com/v0/b/token-library.appspot.com/o/zz.png?alt=media&token=dc866379-92a9-4d85-9173-dad0076046b1                                                                                                              |
+
+  @id1564 @id1436 @tokens
```

Comment:
Also here we will need @mainnet tag (here and in general to be able to separate them next time)